### DISCUSSION TITLE
A32NX_AUTOPILOT_TRACK_SELECTED

### DISCUSSION THREAD

**Discussion by JensKn** [flybywiresim/aircraft#4849] [Help] [answered]
Hi all,

when in TRK/FPA-Mode, I thought LVAR A32NX_AUTOPILOT_TRACK_SELECTED would supply the current track setting. But this LVAR never seems to get a value, at least I am not able to read it (in contrast to other LVARs of A32NX).

Am I missing something or is this a bug?

Thanks!

**Comment by aguther** [comment, marked as answer]
it’s a documentation bug. not used anymore since custom autopilot, see here for an overview:
https://docs.flybywiresim.com/start/autopilot-fbw/

**Reply by JensKn** [reply]
Thanks a lot for clarification!
